integration-cli/docker_cli_run_test.go

```diff
@@ -4519,31 +4433,17 @@ func (s *DockerSuite) TestRunDuplicateMount(c *check.C) {
 	c.Assert(out, checker.Contains, "null")
 }
 
-func (s *DockerSuite) TestRunWindowsWithCPUCount(c *check.C) {
+func (s *DockerSuite) TestRunWindowsWithCPUCountCPUSharesAndCPUPercent(c *check.C) {
 	testRequires(c, DaemonIsWindows)
 
-	out, _ := dockerCmd(c, "run", "--cpu-count=1", "--name", "test", "busybox", "echo", "testing")
+	out, _ := dockerCmd(c, "run", "--cpu-percent=80", "--cpu-count=1", "--cpu-shares=1000", "--name", "test", "busybox", "echo", "testing")
```

Comment:
Can these actually be combined? https://github.com/docker/docker/blob/9fab262690a1e9c465687b3dc22f84b053c5624f/daemon/daemon_windows.go#L89-L128